### PR TITLE
Node performance overview

### PR DESCRIPTION
**Pull request by TApplencourt**
This adds some performance number (and reproducer) for a single node on Aurora.

It's not useful right now (as Aurora "down"), but I don't want to forget I did it, so I'd better pushing it right now! 

Draft and @colleeneb will push the FFT result :)

### CODE REVIEW

**Comment by TApplencourt** [comment]
Ok, so it looks like it's *not* possible to have a link back to the GitHub source code... So I will need to include the source code pf kernels inside the markdown. Not super convenient bug 🤷🏽, no choice it look like.

**Comment by TApplencourt** [comment]
@zippylab @cjknight, Colleen, and I are happy with the result. If you think it's okay, we can merge. We can always update the results when Aurora is back.  Or we can wait and merge it when it's back. I don't have any strong feelings about it.

File: mkdocs.yml [hunk 1]
```diff
@@ -146,6 +146,7 @@ nav:
       - Sunspot to Aurora: aurora/sunspot-to-aurora.md
       - Known Issues: aurora/known-issues.md
       - Hardware Overview: aurora/hardware-overview/machine-overview.md
+      - Node Performance Overvier: aurora/node-performance-overview/node-performance-overview.md
```
Comment (felker) [inline mkdocs.yml:149]: typo
```suggestion
      - Node Performance Overview: aurora/node-performance-overview/node-performance-overview.md
```

**Comment by felker** [comment]
I am coming to this PR with little context, but I am guessing `peer2pear` should be `peer2peer`?

**Comment by TApplencourt** [comment]
For Pear, It was a little joke, but I can fix it!

**Comment by felker** [comment]
@TApplencourt up to you to change it or not, unless @cjknight wants to weigh in. 

Feel free to merge whenever

**Comment by TApplencourt** [comment]
This is ready to merged. Just need on revieqer

**Pull request merged**